l10n_it_fatturapa_in/models/attachment.py

```diff
@@ -124,7 +124,7 @@ def get_invoice_obj(self):
                     att_name=self.display_name,
                     error_msg=e,
                 )
-            _logger.error(error_msg)
```

Comment:
Visto che questo faceva fallire anche i test delle CI OCA, avevo aggiunto vari `mute_logger` per `odoo.addons.l10n_it_fatturapa_in.models.attachment`; puoi rimuoverli visto che ora non servono più? Li puoi trovare tra le modifiche di #2788

Reply:
![image](https://user-images.githubusercontent.com/43987335/177598554-0d7c9770-2058-4b32-b6e3-9465cb6c28e9.png)
intendi questi scusa la banalità e 
![image](https://user-images.githubusercontent.com/43987335/177598666-0ae3599b-d19e-47ec-a1df-81b79f2df4e3.png)

o solo il primo?

Grazie MIlle ancora

Reply:
sennò se mi indichi le righe esatte isolo la cosa in un commit

Grazie ancora dell'aiuto

Reply:
> ![image](https://user-images.githubusercontent.com/43987335/177598554-0d7c9770-2058-4b32-b6e3-9465cb6c28e9.png) intendi questi scusa la banalità e ![image](https://user-images.githubusercontent.com/43987335/177598666-0ae3599b-d19e-47ec-a1df-81b79f2df4e3.png)
> 
> o solo il primo?
> 
> Grazie MIlle ancora

Tutti e tre

Reply:
ok procedo

Reply:
> forse l'ultimo non serve da togliere?

Perché?

Reply:
ho visto che Travis diventa di nuovo rosso

Reply:
> ho visto che Travis diventa di nuovo rosso o così è accettabile?

Travis rosso non è accettabile.
Al momento vedo che sta fallendo per errori di formattazione:
> l10n_it_fatturapa_in/tests/test_import_fatturapa_xml.py:180:17: E126 continuation line over-indented for hanging indent
l10n_it_fatturapa_in/tests/test_import_fatturapa_xml.py:782:17: E126 continuation line over-indented for hanging indent
l10n_it_fatturapa_in/tests/test_import_fatturapa_xml.py:784:13: E121 continuation line under-indented for hanging indent

(da https://app.travis-ci.com/github/OCA/l10n-italy/jobs/575930923#L333-L335)

Reply:
puoi dirmi dove è l'errore di formattazione ho riguardo il codice e mi sfugge.
Grazie

Reply:
> puoi dirmi dove è l'errore di formattazione ho riguardo il codice e mi sfugge. Grazie

Ti avevo già riportato gli errori, sono:
> > l10n_it_fatturapa_in/tests/test_import_fatturapa_xml.py:180:17: E126 continuation line over-indented for hanging indent
l10n_it_fatturapa_in/tests/test_import_fatturapa_xml.py:782:17: E126 continuation line over-indented for hanging indent
l10n_it_fatturapa_in/tests/test_import_fatturapa_xml.py:784:13: E121 continuation line under-indented for hanging indent

Ti aiuto a interpretare il primo, gli altri sono analoghi.

> l10n_it_fatturapa_in/tests/test_import_fatturapa_xml.py:180:17: E126 continuation line over-indented for hanging indent

Vuol dire che nel file
> l10n_it_fatturapa_in/tests/test_import_fatturapa_xml.py

alla riga 180, colonna 17, c'è un errore di indentazione.
Dalle modifiche della PR vedo che in quella posizione c'è:
![image](https://user-images.githubusercontent.com/101628000/178443228-497335ba-dc5b-43be-89da-f876dc069774.png)
Quindi dopo le modifiche, il codice è:
```
        attachment = self.create_attachment(
                'test6_orm_exception', 'IT05979361218_fake.xml.p7m')
```
quindi la riga `'test6...` ha due indentazioni più della riga `attachment...`, da qui l'errore: per correggere suppongo ti basti rimuovere un'indentazione dalla riga `'test6...`.

Reply:
Grazie del chiarimento provo subito, non l'avevo capito a parte leggere l'errore che mi linkavi

Reply:
@SirTakobi  ora ci dovremmo essere

Reply:
devo unire gli ultimi due commit al fine di avere il merge?

Reply:
> devo unire gli ultimi due commit al fine di avere il merge?

I commit "fix test nn" non credo abbiano senso da soli perché correggono un errore introdotto da un commit precedente, "remove mute logger", creato da questa PR: puoi unirli?

Anche "[FIX][l10n_it_fatturapa_in] remove mute logger" potrebbe essere schiacciato insieme alle altre modifiche di l10n_it_fatturapa_in.

Idealmente, secondo me, da questa PR potrebbero uscire esattamente due commit: uno per ogni modulo modificato.

Reply:
fatto